src/api/app/models/user.rb

```diff
@@ -984,6 +984,11 @@ def update_notifications(params)
     User.update_notifications(params, self)
   end
 
+  def is_maintainer_of_project?(project_name)
+    project = Project.find_by_name(project_name)
+    project && project.maintainers.include?(self)
+  end
```

Comment:
This and the maintainer functions duplicate project.user_has_role?(self, 'maintainer')